projects/lib/src/board/westernboard.cpp

```diff
@@ -94,6 +95,13 @@ void WesternBoard::vInitialize()
 	m_rookOffsets[1] = -1;
 	m_rookOffsets[2] = 1;
 	m_rookOffsets[3] = m_arwidth;
+
+	m_pawnAmbiguous = pawnAmbiguity(StepType::Free);
+}
+
+inline int WesternBoard::getPawnPush(const PawnStep& ps, int sign) const
```

Comment:
I am sorry, I have not refreshed the page and so I did not  see your answer when I cited iscocpp.org.... I must remember to refresh the pages. Sometime I see automatic updates to discussions and sometimes not.